nx-X11/lib/X11/Imakefile

```diff
@@ -891,6 +891,13 @@ OBJS3 = \
 	xim_trans.o \
 	$(XKBOBJS) $(LCOBJS)
 
+SHAREDLCDEPSYMSRCS = XlcDL.c XlcSL.c
+
+SHAREDI18NSTATICSRCS = \
+        XDefaultIMIF.c XDefaultOMIF.c
+
+STATICLCDEPSYMSRCS =
```

Comment:
??? Looks like I have done something wrong. This revert belongs to my static stuff: https://github.com/uli42/nx-libs/tree/nxagent_static

I'll update this PR accordingly